test/SlowTests/Issues/RavenDB_11906.cs

```diff
@@ -129,7 +129,6 @@ public void SupportForCreateFieldWithOptions(Options options)
 
         [Theory]
         [RavenData(SearchEngineMode = RavenSearchEngineMode.Lucene)]
```

Comment:
`[RavenData(SearchEngineMode = RavenSearchEngineMode.All)]`

Reply:
No, dynamic fields are not supported yet.